.github/workflows/checks.yml

```diff
@@ -75,3 +75,31 @@ jobs:
 
       - run: snforge --version | grep "snforge 0.16.0"
         shell: bash
+
+  test-action-with-fixed-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Setup Scarb"
+        uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-version: 2.9.4
+
+      - name: "Setup foundry using fixed version"
+        uses: ./
+        with:
+          starknet-foundry-version: 0.44.0
```

Comment:
nitpick: please use older version (right now this is latest, so it does not test what you want to test, it may have fallen back to latest silently)

Reply:
Latest is 0.45.0 now